cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -19,6 +19,8 @@ import (
 
 	"github.com/google/zoekt"
 	"github.com/google/zoekt/build"
+
+	sglog "github.com/sourcegraph/log"
```

Comment:
we should aim for all uses of log to be replace with sglog so we can just import it as log. Fine to be follow up PR though

Reply:
Agreed, I believe this will happen as we incrementally replace all log package dependencies across zoekt